etc/ibm-cloud-sdk-core.api.md

```diff
@@ -0,0 +1,458 @@
+## API Report File for "ibm-cloud-sdk-core"
```

Comment:
Perhaps this one also should not be tracked in git???

Reply:
This file is supposed to be tracked https://api-extractor.com/pages/overview/demo_api_report/
It contains the description of the public API surface. Tracking it allows you to detect intended or unintended changes to the public API.